### PR TITLE
Update jQuery.validate library

### PR DESCRIPTION
**Pull request by eduard13**
### Description (*)
This PR aims to update the jQuery.validate library to the latest `1.19.3` version and keep supporting the current version of jQuery.metadata library.

### Related Pull Requests
https://github.com/magento/partners-magento2b2b/pull/583

### Fixed Issues (if relevant)

1. Fixes magento/magento2#33853

### Manual testing scenarios (*)
N/A

### Questions or comments
ℹ️ For compatibility reasons with the existing code, for now we have to keep the jQuery.metadata library and the rest of adjustments that were made into this library.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by eduard13** [comment]
@magento run all tests

File: lib/web/jquery/jquery.validate.js [hunk 1]
```diff
@@ -296,599 +378,894 @@
             dateISO: "Please enter a valid date (ISO).",
             number: "Please enter a valid number.",
             digits: "Please enter only digits.",
-            creditcard: "Please enter a valid credit card number.",
```
Comment (ihor-sviziev) [inline lib/web/jquery/jquery.validate.js:299]: I think removing credit card that’s something not expected

**Comment by eduard13** [comment]
@magento run all tests

**Comment by eduard13** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Static Tests

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Please fix static test failure

**Comment by eduard13** [comment]
@magento run Static Tests

**Comment by eduard13** [comment]
@magento run Static Tests

**Comment by ihor-sviziev** [comment]
Static tests are passing
@magento run all test

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by eduard13** [comment]
@magento run all tests

**Review by andrewbess — APPROVED** [review]
Hello @eduard13 
Thank you for your contribution.
Changes approved.
Waiting for builds

**Review by ihor-sviziev — APPROVED** [review]

**Pull request merged**